lib/internal/event_target.js

```diff
@@ -347,6 +347,13 @@ function weakListeners() {
   return { registry: weakListenersState, map: objectToWeakListenerMap };
 }
 
+const FLAG_ONCE = 1;
```

Comment:
constants in node start with a k and then follow camel case so this would be 
```suggestion
const kFlagOnce = 1;
```

It would also be preferable to use explicit bit shifts instead of manual powers of two, e.g. 

```suggestion
const kFlagOnce  = 1 << 0;
const kFlagCapture  = 1 << 2;
const kFlagPassive  = 1 << 2;
```


etc